packages/framework/build.xml

```diff
@@ -679,6 +831,23 @@
 
     <target name="product-search-recreate-structure" depends="product-search-delete-structure,product-search-create-structure" description="Recreates structure for searching via elasticsearch (deletes existing structure and creates new one)"/>
 
+    <target name="production-protection" depends="environment-load" description="Asks user if wants to run desctructive actions in production environemnt" hidden="true">
+        <if>
+            <and>
+                <not><isset property="productionConfirmDestructiveAction"/></not>
```

Comment:
we do not use camelCase for phing properties, however, I am not sure, what should be the proper name here, maybe just `production.confirm.destructive.action`?

Reply:
fixed